### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by coopbri** [graphql-nexus/nexus#1154] [General]
Just wanted to say thank you for picking up maintenance on Nexus and breathing new life into it.

After trying out other schema generation libraries (including Pothos, `gqtx`, and others) and giving them an honest shot for months each, I circled back to Nexus and couldn't be happier, it's my favorite! No hate to those libraries, of course, they're awesome. I am just grateful and wanted to express that.

**Comment by Amaan215422** [comment]
Keep growing mate
